src/point.js

```diff
@@ -149,6 +149,18 @@ $.Point.prototype = {
     },
 
     /**
+     * Compute the squared distance between this point and another point.
+     * Useful for optimizing things like comparing distances.
+     * @function
+     * @param {OpenSeadragon.Point} point The point to compute the distance with.
+     * @returns {Number} The distance between the 2 points
```

Comment:
The important bit in this comment is "comparison". For purposes of comparison, squared distance is the same as regular distance - since all we care about is relative values, any monotonic transformations applied to both sides equally will not change the result.

I tried to make this clear in the comments, but didn't want to drop in a long explanation or proof. Any suggestions on a better comment?